src/execution/sql/ddl_executors.cpp

```diff
@@ -156,4 +156,10 @@ bool DDLExecutors::CreateIndex(const common::ManagedPointer<catalog::CatalogAcce
   TERRIER_ASSERT(result, "CreateIndex succeeded, SetIndexPointer must also succeed.");
   return true;
 }
+
+bool DDLExecutors::CreateViewExecutor(common::ManagedPointer<planner::CreateViewPlanNode> node,
+                                      common::ManagedPointer<catalog::CatalogAccessor> accessor) {
+  // const auto table_oid = accessor->CreateView(node->GetNamespaceOid(), node->GetViewName(), *(node->GetSchema()));
```

Comment:
Why is this returning true always?.

Reply:
Just a simple definition requiring further implementation.